utils/lib/rr_graph/graph2.py

```diff
@@ -28,48 +36,128 @@ class NodeType(Enum):
 
 
 class NodeDirection(Enum):
+    """VPR Node Direction. This is a superset of Direction in channel.py
+    See: https://vtr-verilog-to-routing.readthedocs.io/en/latest/vpr/file_formats.html#tag-nodes-node
+    """  # noqa: E501
     NO_DIR = 0
     INC_DIR = 1
     DEC_DIR = 2
     BI_DIR = 3
 
 
 class PinType(Enum):
+    """ Enum for PinClass type
```

Comment:
Should there be a space after ``"""``?  I see both styles here.  I assume yapf doesn't care?